src/main/java/org/cryptomator/ui/fxapp/FxApplication.java

```diff
@@ -75,6 +76,23 @@ public void start() {
 			appWindows.checkAndShowUpdateReminderWindow();
 		}
 
+		var dokany = "org.cryptomator.frontend.dokany.mount.DokanyMountProvider";
```

Comment:
sine this is more an ID for the dokany mount service, i suggest to rename it to `dokanProviderId`.